app/src/androidTest/java/com/android/sample/model/userAccount/UserAccountLocalCacheTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.android.sample.model.userAccount
+
+import android.content.Context
+import androidx.test.core.app.ApplicationProvider
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.test.runTest
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.`is`
+import org.hamcrest.Matchers.nullValue
+import org.junit.Before
+import org.junit.Test
+
+class UserAccountLocalCacheTest {
```

Comment:
Simple and effective test. It's good to know for those who did not touch the cache part of the project how it works, and this test helps !